setup.cfg

```diff
@@ -0,0 +1,9 @@
+[isort]
+line_length = 120
+known_third_party = ddt,mock
+known_django = django
+known_djangoapp = jsonfield,model_utils
+known_edx =
```

Comment:
Yeah, it has something to do with isort configuration. I copied this over from https://github.com/edx/edx-celeryutils/blob/master/setup.cfg, I think we can remove the `known_edx` line